ads-txt.php

```diff
@@ -31,7 +31,16 @@ function tenup_display_ads_txt() {
 		if ( ! empty( $post_id ) ) {
 			$post = get_post( $post_id );
 			header( 'Content-Type: text/plain' );
-			echo esc_html( $post->post_content );
+			$adstxt = esc_html( $post->post_content );
+
+			/**
+			 * Filter the ads.txt content.
+			 *
+			 * @since x.x.x
+			 *
+			 * @param type  $adstxt The existing ads.txt content.
+			 */
+			echo apply_filters( 'ads_txt_content', $adstxt );
```

Comment:
can we apply the `esc_html` here at the point of output (and remove it above). For security reasons, we always want to **escape on output**.